_opensearch/rest-api/snapshots/get-snapshot.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: default
+title: Get Snapshot
+parent: Snapshot APIs
+grand_parent: REST API reference
+nav_order: 4
+---
+
+## Get snapshot.
+
+Retrieves information about a snapshot.
+
+### Path parameters
+
+| Parameter | Data Type | Description |
+| :--- | :--- | :--- |
+| repository | String | The repository that contains the snapshot to retrieve. |
+| snapshot | Snapsnot | Snapshot to retrieve.
+
+### Query parameters
+
+| Parameter | Data Type | Description | 
+:--- | :--- | :---
+| verbose | Boolean | Whether to show all, or just basic snapshot information. If `true`, returns all information. If `false`, omits information like start/end times, failures, and shards. Optional, defaults to `true`.|
+| ignore_unavailable | Boolean | How to handle snapshots that are unavailable (corrupted or otherwise temporarily can't be returned). If `true` and the snapshot is unavailable, the request does not return the snapshot. If `false` and the snapshot is unavailable, the request returns an error. Optional, defaults to `false`.|
+
+#### Sample request
+
+The following request retrieves information for the `my-first-snapshot` located in the `my-opensearch-repo` repository:
+
+````json
+GET _snapshot/my-opensearch-repo/my-first-snapshot
+````
+
+#### Sample response
+
+Upon success, the response returns snapshot information:
+
+````json
+{
+  "snapshots" : [
+    {
+      "snapshot" : "my-first-snapshot",
+      "uuid" : "3P7Qa-M8RU6l16Od5n7Lxg",
+      "version_id" : 136217927,
+      "version" : "2.0.1",
+      "indices" : [
+        ".opensearch-observability",
+        ".opendistro-reports-instances",
+        ".opensearch-notifications-config",
+        "shakespeare",
+        ".opendistro-reports-definitions",
+        "opensearch_dashboards_sample_data_flights",
+        ".kibana_1"
+      ],
+      "data_streams" : [ ],
+      "include_global_state" : true,
+      "state" : "SUCCESS",
+      "start_time" : "2022-08-11T20:30:00.399Z",
+      "start_time_in_millis" : 1660249800399,
+      "end_time" : "2022-08-11T20:30:14.851Z",
+      "end_time_in_millis" : 1660249814851,
+      "duration_in_millis" : 14452,
+      "failures" : [ ],
+      "shards" : {
+        "total" : 7,
+        "failed" : 0,
+        "successful" : 7
+      }
+    }
+  ]
+}
+````
+
```

Comment:
Add a heading "Response body fields"
Field | Data Type | Description

Reply:
I have made this change.

Reply:
Please change this heading to "Response fields"